app/components/claim/card_component.rb

```diff
@@ -0,0 +1,9 @@
+class Claim::CardComponent < ApplicationComponent
```

Comment:
Does this need a spec and a preview?

Reply:
Ah yes, I'll add that. Nice catch.

Reply:
Added a relatively basic spec for the component, along with a preview.